tools/config.py

```diff
@@ -10,10 +10,15 @@
 def init_config():
     config = ConfigParser.ConfigParser()
     config.add_section("GitHub")
+    config.set("GitHub", "username", "")
```

Comment:
You don't need username in config file because with token you got username and email...